opensrp-maternity/src/main/java/org/smartregister/maternity/presenter/MaternityProfileActivityPresenter.java

```diff
@@ -172,7 +172,7 @@ public void startFormActivity(@NonNull String formName, @NonNull String caseId,
 
                 // Fetch saved form & continue editing
                 if (formName.equals(MaternityConstants.Form.MATERNITY_OUTCOME) || formName.equals(MaternityConstants.Form.MATERNITY_MEDIC_INFO)) {
-                    mProfileInteractor.fetchSavedPartialForm(caseId, entityTable);
+                    mProfileInteractor.fetchSavedPartialForm(MaternityUtils.convertFormNameToType(formName), caseId, entityTable);
```

Comment:
Here you can call `form.optString(JsonFormConstants.ENCOUNTER_TYPE)` instead of `MaternityUtils.convertFormNameToType(formName)`

Reply:
You should be fetching with form type not even type